scss/_patterns_tooltips.scss

```diff
@@ -75,6 +76,12 @@ $tooltip-overlay-top-border-radius: 0% 0% 100% 100%;
     }
   }
 
+  .p-tooltip--detached {
+    display: inline;
```

Comment:
Not sure why it's `display: inline` - tooltip seems to be a block element, so `block` seems to be more appropriate.